README.md

```diff
@@ -948,7 +948,7 @@ Note: The included CCI to NIST Mappings are the extracted from NIST.gov, for map
 convert ckl2POAM              Translate DISA Checklist CKL file(s) to POA&M files
 
   USAGE
-    $ saf convert ckl2POAM -i <disa-checklist> -o <poam-output-folder> [-h] [-O <office/org>] [-d <device-name>] [-s <num-rows>]
+    $ saf convert ckl2POAM -i <disa-checklist>... -o <poam-output-folder> [-h] [-O <office/org>] [-d <device-name>] [-s <num-rows>]
```

Comment:
Did you add the CAAT convert info to the README?

Reply:
needed to get feedback from eugene where he wanted it in the order since the hdf2blah conversions are intentionally not in alphabetical order